### PR TITLE
DOCSP-38360 filling out snippets reference page

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Adding text to the snippets reference page 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/mongodb-shell/DOCSP-38360/snippets/reference/

## JIRA

https://jira.mongodb.org/browse/DOCSP-38360

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=664e2c6ccf5bec1c29c0bcd2

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/snippets/reference.txt [hunk 1]
```diff
@@ -12,6 +12,20 @@ Snippets Reference
...
+:ref:`snip-commands`
+   An overview of the commands that are available to work with
+   :ref:`snippets <snip-overview>`.
+
+:ref:`snip-configure`
+   Options to control the interaction between ``mongosh`` and the
+   package manager that tracks individual snippet packages.
+
+:ref:`snip-errors`
+   Expressions to catch runtime errors and display custom error messages.
```
Comment (nvillahermosa-mdb) [inline source/snippets/reference.txt:24]: The handler / matcher itself isn't the regular expression. I'd copy the shortdesc from the linked topic.

"Error handlers allow you to specify regular expressions to catch runtime errors and display custom error messages."

**Review by nvillahermosa-mdb — COMMENTED** [review]
@ltran-mdb2 most of this looks good but I left a comment for one entry.

File: source/snippets/reference.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/snippets/reference.txt]: changed the wording (but left as a fragment like the other ToC items). What do you think?

**Review by nvillahermosa-mdb — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**